index.php

```diff
@@ -27,14 +27,14 @@
 		<meta name="date" content="2010-01-01" />
 		<meta name="page-topic" content="OpenRailwayMap" />
 		<meta name="robots" content="index,follow" />
-		<link rel="alternate" type="application/rss+xml" title="OpenRailwayMap RSS Feed" href="http://blog.openrailwaymap.org/<?php echo $lang; ?>.rss" />
+		<link rel="alternate" type="application/rss+xml" title="OpenRailwayMap RSS Feed" href="https://blog.openrailwaymap.org/<?php echo $lang; ?>.rss" />
```

Comment:
strictly speaking this would belong in the first commit, as well as the next 2

Reply:
???

Reply:
I moved it to another commit and squashed them.